managers/templates/managers/project_form.html

```diff
@@ -33,15 +35,14 @@ <h2>
   {% trans 'Do you want to delete' %} <br/>
   <b>{{object.name}}</b> {% trans 'project' %}?
 </div>
+
+<link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/smoothness/jquery-ui.css">
```

Comment:
This should be placed in `{% block extra_head %}` above `{% block content %}`

Reply:
After fixing the issues in **base.html** does this still apply or should it all be moved to the `{% block extra_script %}`?

Reply:
css'es should be in `{% block extra_head %}` which should be located at the end of `<head>` in `base.html`.